install-odoo-saas.sh

```diff
@@ -158,8 +158,9 @@
          apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false -o APT::AutoRemove::SuggestsImportant=false npm
          rm -rf /var/lib/apt/lists/* wkhtmltox.deb
      fi
-
-     apt-get install -y adduser node-less node-clean-css python python-dateutil python-decorator python-docutils python-feedparser python-imaging python-jinja2 python-ldap python-libxslt1 python-lxml python-mako python-mock python-openid python-passlib python-psutil python-psycopg2 python-babel python-pychart python-pydot python-pyparsing python-pypdf python-reportlab python-requests python-suds python-tz python-vatnumber python-vobject python-werkzeug python-xlwt python-yaml
+     
+     apt-get update -y
```

Comment:
THere is ``apt-get update`` in Dockerfile already. Also, ``apt-get update`` doesn't work properly sometimes and has to be repeated.
For these reason, I don't include it in the script

Reply:
Tested at multiple times, without it, it doesn't install the following python modules and stop the script.

Reply:
Yes, this is why README requires run ``apt-get update`` *before* running the script

Reply:
With Docker, exactly and in this case it's working.
But with the "basic usage" script, the one tested for this request, it create errors.

Reply:
*Basic usage* requires to run ``apt-get update`` manually before running the script

Reply:
Oh! I dit it but will retry.
Also the second problem with the line "apt-get install -y adduser", is that python-pypdf doens't exist on ubuntu 17.04

Reply:
Re-tested, apt-get in fact well made before running the script, but always problems while 'E: Unable to locate package python-'

Reply:
@E-HO it doesn't matter do you run apt-get inside script or outside. 
But when you do it outside, you can repeat it, while script run it only once.

From README:

    apt-get update | grep "Hit http\|Ign http" -C 10000 && echo "There are possible failures on fetching. Try apt-get update again"